oak_runtime/src/introspect.rs

```diff
@@ -25,6 +25,71 @@ use prost::Message;
 use regex::Regex;
 use std::{net::SocketAddr, sync::Arc};
 
+#[cfg(not(feature = "oak_introspection_client"))]
+mod introspection_client {
+    use super::*;
+
+    pub fn find_client_file(_path: &str) -> Option<Response<Body>> {
+        None
+    }
+}
+
+#[cfg(feature = "oak_introspection_client")]
+mod introspection_client {
+    use super::*;
+    use hyper::header::CONTENT_ENCODING;
+
+    fn static_file(
+        file: &str,
+        content_type: &str,
+        content_encoding: Option<&str>,
```

Comment:
Maybe create a bool-like enum for this, e.g.

```rust
enum Compress {
  Yes,
  No,
}
```

which also helps with readability when invoking the function.

Reply:
Mmmh, not sure if it makes sense to abstract the HTTP header with a boolean. Since compression is a property of the file (as opposed to compression being applied by the rust code) I think it makes sense to explicitly specify which type was used. Theoretically this might differ per file as well, with other compressions such as brotli existing.

Reply:
The readability concern imho will be addressed by the comment explaining what headers do exactly.